frontend/src/views/Dashboard.js

```diff
@@ -2,6 +2,8 @@ import styles from '../styles/Dashboard.module.css';
 import { PassageAuthGuard } from '@passageidentity/passage-react';
 import { usePassageUserInfo } from '../hooks/';
 import LogoutButton from '../components/LogoutButton';
+import { BrowserRouter, Routes, Route, Outlet, Link } from 'react-router-dom';
+import FriendGroup from './FriendGroup';
```

Comment:
I don't know if we're supposed to be using the `Dashboard.js` file or if we're supposed to create a new view. The `Dashboard.js` is part of the Passage authentication I think... idk.

Reply:
The Dashboard.js originally indicated that it was the homepage so I was trying to modify it to look like our homepage.  After the routing and link functionality is established, I can work on the css styling.